src/chrome/content/rules/TYPO3.xml

```diff
@@ -1,114 +1,25 @@
 <!--
-	For rules causing false/broken MCB, see TYPO3.org-falsemixed.xml.
-
-	Other TYPO3 rulesets:
-
-		- T3Blog.com.xml
-
-
 	Nonfunctional hosts in *typo3.org:
-
-		- demo ¹
-		- lists ²
-
-	¹ Dropped
-	² Refused
-
-
-	Problematic hosts in *typo3.org:
-
-		- flow ¹
-		- neos ²
-
-	¹ Mixed css from $self
-	² Mismatched
-
-
-	Insecure cookies are set for these domains and hosts:
-
-		- .typo3.org
-		- buzz.typo3.org
-		- forge.typo3.org
-		- .forum.typo3.org
-
-
-	Mixed content:
-
-		- css on flow from $self *
-		- Images flow from $self *
-		- favicon on buzz, flow from $self *
-		- Bug on ^, buzz, forum, wiki from piwik.typo3.org *
-
-	⁴ Secured by us
-
+		- lists
 -->
 <ruleset name="TYPO3.org (partial)">
 
 	<!--	Direct rewrites:
 				-->
 	<target host="typo3.org"/>
-	<target host="association.typo3.org" />
 	<target host="buzz.typo3.org" />
-	<target host="certification.typo3.org" />
```

Comment:
The DNS records for association and certification are still alive, so please don't remove them.